CondTools/BeamSpot/plugins/BeamSpotOnlineRecordsWriter.cc

```diff
@@ -136,11 +138,21 @@ void BeamSpotOnlineRecordsWriter::endJob() {
 
   lastAnalyzedFill = -999;
 
+  // Verify that the parsing was correct by checking the BS positions
+  if (std::fabs(x) > 1000. || std::fabs(x) < 1.e-20 || std::fabs(y) > 1000. || std::fabs(y) < 1.e-20 ||
+      std::fabs(z) > 1000. || std::fabs(z) < 1.e-20) {
+    edm::LogError("BeamSpotOnlineRecordsWriter")
+        << " !!! Error in parsing file, parsed BS (x,y,z): (" << x << "," << y << "," << z << ") !!!";
```

Comment:
What about throwing instead?

Reply:
I put a `return` statement after this, so it won't return anythin and print the LogError message. 
What is the advantage of throwing?

Reply:
Functionally none, but it would be harder to ignore.

Reply:
Replace `LogError` with `throw` in e5a659f